src/device/onLoad.js

```diff
@@ -74,6 +74,12 @@ realityEditor.device.onload = async function () {
     if (globalStates.platform !== 'iPad' && globalStates.platform !== 'iPhone' && globalStates.platform !== 'iPod touch') {
         globalStates.platform = false;
     }
+    
+    if (realityEditor.device.environment.isSafari()) {
```

Comment:
This should have a comment on why ("iframes in safari intercept pointer-events if not given a negative z-index") it's done